src/app/frontend/common/components/volumemount/template.html

```diff
@@ -0,0 +1,94 @@
+<!--
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<kd-card [initialized]="initialized"
```

Comment:
Here is the nested card. It adds multiple styles, including the shadow.

Reply:
Sorry, but I really really really really suck at CSS. Could you guys help me here?

Also, I don't think a container specific card is a good idea. 99% of the pods have only one container. In the end of the day people use `deployments` and `pods` and the replica set between them are already an extra annoying click. Adding a second one, although logically correct, would make things less pratical.

Reply:
I can fix this, @floreks probably too. Let's go back to it on Thursday.

/assign @floreks @maciaszczykm

Reply:
Thank you for working on the main part. We'll update styling on Thursday to improve the UX here and it should be ready.

Reply:
Yeah, you have done very good job here. Styling is the last part that should be pretty quick right now. Thanks :)